src/legacy/ui/public/vislib/components/zero_injection/zero_fill_data_array.js

```diff
@@ -19,34 +19,31 @@
 
 import _ from 'lodash';
 
-export function VislibComponentsZeroInjectionZeroFillDataArrayProvider() {
-
-  /*
-   * Accepts an array of zero-filled y value objects (arr1)
-   * and a kibana data.series[i].values array of objects (arr2).
-   * Return a zero-filled array of objects (arr1).
-   */
-
-  return function (arr1, arr2) {
-    if (!Array.isArray(arr1) || !Array.isArray(arr2)) {
-      throw new TypeError('ZeroFillDataArrayUtilService expects 2 arrays');
-    }
-
-    let i;
-    let val;
-    let index;
-    const max = arr2.length;
-
-    const getX = function (d) {
-      return d.x === val.x;
-    };
+/*
+  * Accepts an array of zero-filled y value objects (arr1)
+  * and a kibana data.series[i].values array of objects (arr2).
+  * Return a zero-filled array of objects (arr1).
+  */
+
+export function zeroFillDataArray(arr1, arr2) {
+  if (!Array.isArray(arr1) || !Array.isArray(arr2)) {
+    throw new TypeError('ZeroFillDataArrayUtilService expects 2 arrays');
```

Comment:
It's not an angular Service anymore, so you could update the error message

Reply:
solved in 0a9c70ab94